pkg/operator/operator/errors.go

```diff
@@ -260,4 +262,20 @@ func ErrorAPINotDeployed(apiName string) error {
 		Kind:    ErrAPINotDeployed,
 		Message: fmt.Sprintf("%s is not deployed", apiName), // note: if modifying this string, search the codebase for it and change all occurrences
 	})
+
+}
+
+func ErrorRegistryAccountIDMismatch(regID, opID string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrRegistryAccountIDMismatch,
+		Message: fmt.Sprintf("AWS registry account ID %s doesn't match operator's account ID %s", regID, opID),
+	})
+}
+
+func ErrorDockerImageInaccessible(image string, cause error) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrDockerImageInaccessible,
+		Message: fmt.Sprintf("%s is not accessible", image),
```

Comment:
the error message of the original error should be included in `Message`, here is [an example](https://github.com/cortexlabs/cortex/blob/master/pkg/lib/aws/errors.go#L70) (the only difference is here you probably want more than the first line, so you can use `errors.Message(err)` to get the whole thing (that helper pretty much just makes sure there's no leading/trailing whitespace)

Reply:
Agreed. Keeping it consistent with all other error functions makes sense.